src/servers/udp/server.rs

```diff
@@ -252,124 +329,178 @@ impl Udp {
         tx_start: oneshot::Sender<Started>,
         rx_halt: oneshot::Receiver<Halted>,
     ) {
-        let socket = Arc::new(
-            UdpSocket::bind(bind_to)
-                .await
-                .unwrap_or_else(|_| panic!("Could not bind to {bind_to}.")),
-        );
-        let address = socket
-            .local_addr()
-            .unwrap_or_else(|_| panic!("Could not get local_addr from {bind_to}."));
-        let halt = shutdown_signal_with_message(rx_halt, format!("Halting Http Service Bound to Socket: {address}"));
-
-        info!(target: "UDP TRACKER", "Starting on: udp://{}", address);
-
-        let running = tokio::task::spawn(async move {
-            debug!(target: "UDP TRACKER", "Started: Waiting for packets on socket address: udp://{address} ...");
-            Self::run_udp_server(tracker, socket).await;
-        });
+        let halt_task = tokio::task::spawn(shutdown_signal_with_message(
+            rx_halt,
+            format!("Halting Http Service Bound to Socket: {bind_to}"),
```

Comment:
Hi @da2ce7 I think the message should be "Halting UDP Service Bound to Socket: {bind_to}"